WinUIGallery/Samples/ControlPages/Design/TypographyPage.xaml

```diff
@@ -39,7 +39,7 @@
         </Grid.RowDefinitions>
         <RichTextBlock>
             <Paragraph>
-                Typography helps provide structure and hierarchy to UI. The default font for Windows is <Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>
+                Typography helps provide structure and hierarchy to UI. The default font for Windows is<Hyperlink NavigateUri="https://learn.microsoft.com/windows/apps/design/downloads/#fonts">Segoe UI Variable</Hyperlink>
```

Comment:
Naughty XAML styler.

Reply:
I hate this 'bug' with a passion 😕.

Good catch, fixed!